### ISSUE TITLE
KaTrain crashes when loading SGF file

### ISSUE THREAD

**Issue by sflinter** [sanderland/katrain#70]
(First off, thanks for a great application!)

When I try to load an SGF file into KaTrain, the application crashes.

When I debug the loading, I get the following error:

```
'NoneType' object has no attribute 'strip'
Uncaught exception. Entering post mortem debugging
Running 'cont' or 'step' will restart the program
> /Users/steveflinter/src/katrain/katrain/core/sgf_parser.py(325)_parse_branch()
-> property, value = match[1], match[2].strip()[1:-1]
```

I'm running on a Macbook Pro (Catalina), using python 3.7.4 (miniconda). The AI engine is katago 1.4.2. I'm using the 1.1.2 version of KaTrain:

```
commit dc22aa88526fb6446f908259f06020d649a2d0a9 (HEAD -> master, tag: v1.1.2, origin/master, origin/HEAD)
```

Here's the SGF file (generated by Dragon Go Server):

```
(
;FF[4]GM[1]
AP[DGS:1.20.5]
PC[Dragon Go Server: https://www.dragongoserver.net/]
DT[2020-06-02,2020-06-15]
GN[sflinter-Bokaiido-1321916-20200615]
SO[https://www.dragongoserver.net/game.php?gid=1321916]
PB[Emmanuel Volte (Bokaiido)]
PW[Steve Flinter (sflinter)]
BR[2k]
WR[1k]
XM[207]
GC[Game ID: 1321916
Game Type: GO (1:1)
Rated: Y

White Start Rating: 1k (+4%) - Elo rating 2004
Black Start Rating: 2k (-30%) - Elo rating 1869]
OT[30 days with 1 day extra per move]
RU[Japanese]
SZ[19]
KM[0.5]
;B[pd]
;W[dp]
;B[fq]
;W[cn]
;B[oq]
;W[dc]
;B[jp]
;W[qp]
;B[qq]
;W[rq]
;B[rr]
;W[pq]
;B[qr]
;W[op]
;B[pr]
;W[nq]
;B[pp]
;W[po]
;B[rp]
;W[pq]
;B[or]
;W[ro]
;B[sq]
;W[qf]
;B[pp]
;W[qo]
;B[oo]
;W[pq]
;B[np]
;W[on]
;B[pp]
;W[nn]
;B[qh]
;W[of]
;B[nd]
;W[pi]
;B[qm]
;W[pm]
;B[qk]
;W[qi]
;B[pl]
;W[ol]
;B[ok]
;W[nl]
;B[rn]
;W[so]
;B[nk]
;W[mk]
;B[ni]
;W[mj]
;B[oi]
;W[ph]
;B[mi]
;W[pj]
;B[pk]
;W[ln]
;B[lp]
;W[mf]
;B[ki]
;W[qc]
;B[qd]
;W[rd]
;B[pc]
;W[rb]
;B[qb]
;W[rc]
;B[kc]
;W[di]
;B[jh]
;W[jk]
;B[gc]
;W[ce]
;B[fo]
;W[ic]
;B[id]
;W[jd]
;B[hd]
;W[jc]
;B[kd]
;W[je]
;B[lf]
;W[if]
;B[gf]
;W[hg]
;B[fh]
;W[hi]
;B[gg]
;W[dg]
;B[fj]
;W[gi]
;B[fi]
;W[ij]
;B[dj]
;W[cj]
;B[ck]
;W[ci]
;B[dk]
;W[dr]
;B[er]
;W[eq]
;B[fr]
;W[en]
;B[gm]
;W[in]
;B[ho]
;W[me]
;B[ke]
;W[mb]
;B[mc]
;W[kb]
;B[lb]
;W[la]
;B[lc]
;W[nb]
;B[jb]
;W[ka]
;B[ob]
;W[ib]
;B[nc]
;W[kf]
;B[kg]
;W[jf]
;B[le]
;W[mg]
;B[lg]
;W[fb]
;B[rj]
;W[ri]
;B[cq]
;W[dq]
;B[cr]
;W[ds]
;B[bo]
;W[bn]
;B[co]
;W[do]
;B[bs]
;W[ar]
;B[cp]
;W[ap]
;B[ao]
;W[bq]
;B[dm]
;W[dn]
;B[bk]
;W[an]
;B[cd]
;W[de]
;B[dd]
;W[ed]
;B[cc]
;W[be]
;B[ec]
;W[db]
;B[fc]
;W[cb]
;B[ee]
;W[bc]
;B[fd]
;W[bj]
;B[kk]
;W[kl]
;B[jj]
;W[il]
;B[hk]
;W[ik]
;B[ii]
;W[ih]
;B[hj]
;W[ji]
;B[qe]
;W[re]
;B[ii]
;W[gb]
;B[si]
;W[sh]
;B[sj]
;W[rl]
;B[rm]
;W[sn]
;B[sm]
;W[lj]
;B[mh]
;W[io]
;B[ip]
;W[li]
;B[lh]
;W[kj]
;B[ji]
;W[jg]
;B[kh]
;W[qa]
;B[pb]
;W[ko]
;B[eb]
;W[ea]
;B[bd]
;W[ad]
;B[ed]
;W[bb]
;B[br]
;W[fm]
;B[gl]
;W[gn]
;B[hn]
;W[aq]
;B[em]
)

```

**Comment by sanderland** [comment]
Thanks for the bug report, this is fixed in 1.2 and will be released in the next few days.

**Comment by sflinter** [comment]
Great, thanks for the prompt follow-up.
